server/plugin/template.go

```diff
@@ -149,10 +169,24 @@ func init() {
 		`{{template "eventRepoIssueFullLink" .}} - {{.GetIssue.GetTitle}}`,
 	))
 
+	template.Must(masterTemplate.New("labels").Funcs(funcMap).Parse(`
+{{- if .Labels }}
+Labels: {{range $i, $el := .Labels -}}` + "{{- if $i}}, {{end}}[`{{ $el.Name }}`]({{ $.RepositoryURL }}/labels/{{ $el.Name | pathEscape }})" + `{{end -}}
+{{ end -}}
+`))
+
+	template.Must(masterTemplate.New("assignee").Funcs(funcMap).Parse(`
+{{- if .GetAssignee }}
+Assignee: [{{ .GetAssignee.GetLogin }}]({{ .GetAssignee.GetHTMLURL }})
```

Comment:
It's possible that an issue/PR has multiple assignees. Is that taken into account here?

Reply:
Fair point. As I understand `GetAssignee` returns only a single one. I have modified it so that all assignees are